src/classes/CRLP_RollupAccount_TEST.cls

```diff
@@ -169,6 +169,10 @@ private class CRLP_RollupAccount_TEST {
     static testMethod void test_Rollups_Trigger() {
         testRollupsServices(TestType.TestTrigger);
     }
+    /** @description Test the Trigger method of rolling up when reparenting an Opp to a different Account */
+    static testMethod void test_Rollups_ReparentOppAccount() {
```

Comment:
Formatting: Similar to other comment, mentioning that this should be @isTest annotation.

Reply:
True. Same issue as above though.